app/1.0/configuration.md

```diff
@@ -311,10 +311,10 @@ and Kong will be running in that instance (as configured in `nginx-kong.conf`).
 
 ## Serving both a website and your APIs from Kong
 
-A common use case for API providers is to make Kong serve both a website
+A common use case for Service providers is to make Kong serve both a website
```

Comment:
This looks like an inadvertent find-and-replace - I'd suggest that this be left `API`.

Reply:
Ok, undone